site/content/_index.md

```diff
@@ -1,50 +1,34 @@
 ---
 title: Raise Green
 subtitle: 'Crowdfunding a healthy, just, and sustainable future.'
-image: /img/logo_no_text.png
+image: /img/solarpanels.jpg
```

Comment:
This may be a bug on my end, but were you intending this to be a background image or the logo image?

Reply:
A background image. I didn't see solar panels.jpg in the assets.